src/main/scala/zio/rocksdb/transaction/ZTransactionDB.scala

```diff
@@ -0,0 +1,38 @@
+package zio.rocksdb.transaction
+import org.rocksdb.WriteOptions
+import org.{ rocksdb => jrocks }
+import zio.{ UIO, ZLayer, ZManaged }
+
+final class ZTransactionDB private (transactionDB: jrocks.TransactionDB) extends RocksDB.TransactionDBService {
```

Comment:
Same: `TransactionDB`.

Reply:
Already used name — https://github.com/zio/zio-rocksdb/pull/13/files#diff-dda252e0cfc4debfb3205f637def10acR9

Reply:
Renamed to `LiveTransactionDB`